web/src/app/models/job.model.ts

```diff
@@ -39,4 +39,8 @@ export class Job extends Copiable {
 
     if (!strategy) this.strategy = DataCollectionStrategy.PREDEFINED;
   }
+
+  getTasksSorted(): List<Task> | undefined {
```

Comment:
Let's return empty list instead of undefined so that the caller only has one empty state to manage.